peps/pep-0239.rst

```diff
@@ -1,15 +1,26 @@
 PEP: 239
 Title: Adding a Rational Type to Python
-Version: $Revision$
-Last-Modified: $Date$
 Author: Christopher A. Craig <python-pep@ccraig.org>, Moshe Zadka <moshez@zadka.site.co.il>
 Status: Rejected
 Type: Standards Track
-Content-Type: text/x-rst
 Created: 11-Mar-2001
 Python-Version: 2.2
 Post-History: 16-Mar-2001
 
+.. rejected::
+   BDFL Pronouncement:
+
+   This PEP is rejected.  The needs outlined in the rationale section
```

Comment:
```suggestion
   This PEP is rejected.  The needs outlined in the rationale section
```

I could see the necessity for retaining this title if it was speaking in Guido's voice, as some rejection notices do, but this one does not (and instead quotes Guido in the third person) so it seems simpler to me IMO to just leave it out?